op-conductor/README.md

```diff
@@ -93,6 +93,6 @@ There are 2 situations we need to consider.
 
 1. Leadership transfer triggered by raft consensus protocol (network partition, etc)
    1. In this case, a new leader will be elected regardless of its sync status, it could be behind for a few blocks
-   2. The solution is to simple, wait until the elected leader catch up to tip (same as the FSM tip)
+   2. The solution is to simply, wait until the elected leader catch up to tip (same as the FSM tip)
```

Comment:
```suggestion
   2. The solution is to simply wait until the elected leader catch up to tip (same as the FSM tip)
```

Reply:
@smartcontracts  Thanks for catching that as well!